src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs

```diff
@@ -303,73 +372,9 @@ public void EnumerableStopsAtEndWhenEndIsFirstByteOfEmpty()
             Assert.Equal(new[] { 100, 0 }, sizes);
         }
 
-        [Fact]
-        public void SeekEmptySkipDoesNotCrossPastEnd()
-        {
-            var bufferSegment1 = new BufferSegment<byte>(new byte[100]);
-            BufferSegment<byte> bufferSegment2 = bufferSegment1.Append(new byte[0]);
-            BufferSegment<byte> bufferSegment3 = bufferSegment2.Append(new byte[0]);
-            BufferSegment<byte> bufferSegment4 = bufferSegment3.Append(new byte[100]);
+        #endregion
 
-            var buffer = new ReadOnlySequence<byte>(bufferSegment1, 0, bufferSegment2, 0);
-
-            SequencePosition c1 = buffer.GetPosition(100);
-
-            Assert.Equal(0, c1.GetInteger());
-            Assert.Equal(bufferSegment2, c1.GetObject());
-
-            c1 = buffer.GetPosition(100, buffer.Start);
-
-            Assert.Equal(0, c1.GetInteger());
-            Assert.Equal(bufferSegment2, c1.GetObject());
-
-            // Go out of bounds for segment
-            Assert.Throws<ArgumentOutOfRangeException>(() => c1 = buffer.GetPosition(150, buffer.Start));
-            Assert.Throws<ArgumentOutOfRangeException>(() => c1 = buffer.GetPosition(250, buffer.Start));
-        }
-
-        [Fact]
-        public void SeekEmptySkipDoesNotCrossPastEndWithExtraChainedBlocks()
-        {
-            var bufferSegment1 = new BufferSegment<byte>(new byte[100]);
-            BufferSegment<byte> bufferSegment2 = bufferSegment1.Append(new byte[0]);
-            BufferSegment<byte> bufferSegment3 = bufferSegment2.Append(new byte[0]);
-            BufferSegment<byte> bufferSegment4 = bufferSegment3.Append(new byte[100]);
-            BufferSegment<byte> bufferSegment5 = bufferSegment4.Append(new byte[0]);
-            BufferSegment<byte> bufferSegment6 = bufferSegment5.Append(new byte[100]);
-
-            var buffer = new ReadOnlySequence<byte>(bufferSegment1, 0, bufferSegment2, 0);
-
-            SequencePosition c1 = buffer.GetPosition(100);
-
-            Assert.Equal(0, c1.GetInteger());
-            Assert.Equal(bufferSegment2, c1.GetObject());
-
-            c1 = buffer.GetPosition(100, buffer.Start);
-
-            Assert.Equal(0, c1.GetInteger());
-            Assert.Equal(bufferSegment2, c1.GetObject());
-
-            // Go out of bounds for segment
-            Assert.Throws<ArgumentOutOfRangeException>(() => c1 = buffer.GetPosition(150, buffer.Start));
-            Assert.Throws<ArgumentOutOfRangeException>(() => c1 = buffer.GetPosition(250, buffer.Start));
-        }
-
-        [Fact]
-        public void Create_WorksWithArray()
```

Comment:
I think we should keep this test (unless you moved it somewhere else).

Reply:
It is the same as `Ctor_Array_NoOffset()` test. I removed it because it is duplicate of `Ctor_Array_NoOffset()` test.